src/Maestro/tests/scenarios.md

```diff
@@ -0,0 +1,131 @@
+# Maestro++ Scenarios and test coverage
+
+## Overview
+
+This is a list of the business scenarios that Maestro is involved in, and the automated or manual validation to be performed for those scenarios
+
+## Package Publishing
+
+Maestro is only involved in the insertion of builds into the BAR.
+
+### Automated Tests
+Scenario tests for the latest version of the API are available at https://github.com/dotnet/arcade-services/blob/master/src/Maestro/tests/Scenarios/builds.ps1
+
+Unit tests for the 20190116 API version are available at: https://github.com/dotnet/arcade-services/blob/master/src/Maestro/Maestro.Web.Tests/BuildController20190116Tests.cs
+
+## Manual testing
+We don't have scenario or unit tests for older versions of this controller/API. Making POST requests to 
+* https://maestro-int.westus2.cloudapp.azure.com/api/builds/15357?api-version=2018-07-16
+* https://maestro-int.westus2.cloudapp.azure.com/api/builds/15357?api-version=2019-01-16
+
+with the following body:
+```json
+{
+  "repository": "https://github.com/dotnet/arcade",
+  "branch": "riarenas/test",
+  "commit": "12345",
+  "buildNumber": "12",
+  "assets": [
+    {
+      "name": "testasset",
+      "version": "1.0",
+      "nonShipping": true,
+      "locations": [
+        {
+          "location": "alocation",
+          "type": 0
+        }
+      ]
+    }
+  ],
+  "dependencies": [
+  ]
+}
+```
+
+Test that it's possible to publish a build with assets to the BAR for versions not covered by the automated scenario tests.
+
+## Dependency Flow
```

Comment:
Is there a delta?  I think the hope is that with this doc we can not only understand what is currently being tested, but what needs to be manually tested (and automated in the future).  

In other words, I don't see any manual test section here, does that imply we're good?

Reply:
Yes, from my perspective we're good with the scenario tests when it comes to dependency flow